matchms/filtering/remove_spectra_above_intensity.py

```diff
@@ -0,0 +1,30 @@
+from ..typing import SpectrumType
+from . import select_by_relative_intensity
+
+
+def remove_spectra_above_intensity(spectrum: SpectrumType, no_peaks: int = 5,
```

Comment:
```suggestion
def remove_spectra_below_intensity(spectrum: SpectrumType, no_peaks: int = 5,
```

Reply:
Sorry, I realize I am interpreting names probably exactly the opposite way. And in particular here I find it hard to find a good name. ``remove_spectra_few_high_peaks``? ``remove_spectra_few_high_intensity_peaks`` ... ?

Reply:
No worries. The idea here was just to remove spectra that have less than some pre-determined number of peaks above a certain threshold intensity. So I think `remove_spectra_few_high_peaks` is probably the best name here.